parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -87,9 +87,8 @@ fi
 {{end}}
 
 if [[ ${UBUNTU_RELEASE} == "18.04" ]]; then
-  if apt list --installed | grep 'ntp'; then
```

Comment:
In order for this to work on 18.04 prior to building a new VHD w/ chrony already installed is to protect the `configureChrony` and `ensureChrony` invokations inside a `if apt list --installed | grep 'chrony'; then` block.

Otherwise we're going to get a CSE error on all 18.04 VHD nodes (`"distro": "aks-ubuntu-18.04"`):

```
+ sleep 5
+ for i in $(seq 1 $retries)
+ timeout 30 systemctl daemon-reload
+ timeout 30 systemctl restart chrony
Failed to restart chrony.service: Unit chrony.service not found.
+ '[' 100 -eq 100 ']'
+ return 1
+ ret=1
+ systemctl status chrony --no-pager -l
Unit chrony.service could not be found.
+ '[' 1 -ne 0 ']'
+ return 1
+ exit 4
```